### ISSUE TITLE
Boost dependency in relative_pose main.cpp

### ISSUE THREAD

**Issue by jason-holloway** [laurentkneip/opengv#30]
Building in Ubuntu 14.04 using cmake with shared libraries. Ran into the following error:

```
In file included from /home/jason/opengv/src/relative_pose/modules/main.cpp:47:0:
/home/jason/opengv/include/opengv/math/Sturm.hpp:43:32: fatal error: boost/shared_ptr.hpp: No such file or directory
 #include <boost/shared_ptr.hpp>

```
Installing boost solves the problem. You may want to include the dependency in the installation instructions. I didn't check to see if there were any dependencies in other files after getting this build error.

**Comment by laurentkneip** [comment]
I ran into this issue just now by myself, and I am about to fix this. Give it a few hours.

**Comment by laurentkneip** [comment]
This problem is solved by ad138505c00a6d041065dc441bf5365e80ce7dc7
